### ISSUE TITLE
Warning: Undefined array key "precision"

### ISSUE THREAD

**Issue by ameotoko** [isotope/core#2343]
Seems like `DatabaseUpdater` issue. I've got a warning trying to update "Label for blank option" or "Multilingual" for an attribute in back end:

```
ErrorException:
Warning: Undefined array key "precision"

  at vendor/contao/core-bundle/src/Resources/contao/library/Contao/Database/Installer.php:372
  at Contao\Database\Installer->getFromDb()
     (vendor/contao/core-bundle/src/Resources/contao/library/Contao/Database/Installer.php:116)
  at Contao\Database\Installer->compileCommands()
     (vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/DatabaseUpdater.php:30)
  at Isotope\DatabaseUpdater->autoUpdateTables(array('tl_iso_product'))
     (vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/Backend/Attribute/DatabaseUpdate.php:48)
  at Isotope\Backend\Attribute\DatabaseUpdate->onSubmit(object(DC_Table))
     (vendor/contao/core-bundle/src/Resources/contao/drivers/DC_Table.php:2082)
  at Contao\DC_Table->edit()
     (vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/BackendModule/BackendOverview.php:225)
  at Isotope\BackendModule\BackendOverview->getModule('attributes')
     (vendor/isotope/isotope-core/system/modules/isotope/library/Isotope/BackendModule/BackendOverview.php:81)
  at Isotope\BackendModule\BackendOverview->generate()
     (vendor/contao/core-bundle/src/Resources/contao/classes/Backend.php:439)
  at Contao\Backend->getBackendModule('iso_setup', null)
     (vendor/contao/core-bundle/src/Resources/contao/controllers/BackendMain.php:168)
  at Contao\BackendMain->run()
     (vendor/contao/core-bundle/src/Controller/BackendController.php:49)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor/symfony/http-kernel/HttpKernel.php:152)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:74)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:44)
```

**Comment by aschempp** [comment]
are you reporting this to Contao (or do a PR)? 🙃

**Comment by ameotoko** [comment]
After some digging – the actual failing method `Contao\Database\Installer->getFromDb()` is only used by Contao in `Contao\Database\Installer->generateSqlForm()`, which is deprecated since `4.0` and currently not used by Contao itself in any branch. I have doubts my PR will be accepted there 🤔

Maybe it's time to refactor Isotope to use InstallationBundle instead? Or do you insist on a PR to Contao?

**Comment by aschempp** [comment]
hmm, I probably need to dig as well. The install tool will be removed in Contao 5, so I would prefer not to rewrite to it and then change it all over again for the next version 😆 but not sure thats possible at all …

**Comment by ameotoko** [comment]
I could try a PR if you wish.

**Comment by aschempp** [comment]
yes please :)

**Comment by ameotoko** [comment]
Something like #2351, @aschempp? 🙂

**Comment by ameotoko** [comment]
This seems to be fixed now, with Contao `4.13.18` and Isotope `2.8.10`
